occurrence-download/src/main/java/org/gbif/occurrence/download/query/EsQueryVisitor.java

```diff
@@ -152,7 +169,7 @@ private boolean isReplaceableByInPredicate(Predicate predicate, Map<OccurrenceSe
   private List<InPredicate> toInPredicates(Map<OccurrenceSearchParameter, List<EqualsPredicate>> equalPredicates) {
     return equalPredicates.entrySet()
             .stream()
-            .map(e -> new InPredicate(e.getKey(), e.getValue().stream().map(EqualsPredicate::getValue).collect(Collectors.toSet())))
+            .map(e -> new InPredicate(e.getKey(), e.getValue().stream().map(EqualsPredicate::getValue).collect(Collectors.toSet()), false))
```

Comment:
Doesn't this need to pass the verbatimMatch field through, and if it's set on some EqualsPredicates but not others, generate two InPredicates?

Reply:
I fixed that by grouping the predicates by the matchCase field